.agents/skills/iterative-refinement/SKILL.md

```
@@ -0,0 +1,149 @@
+- New features requiring TDD approach
+- Bug fixes needing verification
+
+## Do NOT Use
+
+- Single-file simple changes
+- Documentation-only updates
+- Changes without existing test coverage
+
+## Process
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)